index.html

```diff
@@ -0,0 +1,102 @@
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+  <nav id="navbar">
+    <div class="menu">
+      <a href="#">About</a>
+      <a href="#">Work</a>
+      <a href="#">Contact</a>
+    </div>
+  </nav>
+
+  <section id="welcome-section">
+    <h1>Hey I am Mimic</h1>
+    <p>a web developer</p>
+  </section>
+
+  <!-- project entry -->
+  <section id="projects">
+    <h2>This are some of my projects</h2>
+    <div class="project-container">
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" id="profile-link"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project01"></a>
+        <p>Tribute Page</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" id="profile-link"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project01"></a>
+        <p>Tribute Page</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+    </div>
+  </section>
+
+
+
+  <footer>
+    <div class="social-links">
+      <a href="#"><i class="fa fa-facebook-square"></i> Facebook</a>
+      <a href="#"><i class="fa fa-github"></i> Github</a>
+      <a href="#"><i class="fa fa-twitter"></i> Twitter</a>
+      <a href="#"><i class="fa fa-at"></i> Email</a>
+      <a href="#"><i class="fa fa-mobile"></i> Phone</a>
+    </div>
+
+    <div class="bottom-footer">
+      
+    </div>
+  </footer>
+
+
+
+
+
+
+
+
```

Comment:
- [ ] Good eye 👁️ catching the problem however please ☺️  always phrase the comments politely 🙏 , as suggestions instead of orders 🔨 . For example:
   * This is a polite suggestion: *"I think it would be better to use semantic tags"*
   * This reads as an order: *"Use semantic tags"*
  
In general, it is important to be courteous and respectful while also being very clear and helpful to the developer whose code you are reviewing. Using a polite tune while suggesting is very important if you want your code review to be taken as positive feedback instead of criticism. Here is a link to [the part of the lesson about communication](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) that you can use to enhance your communication 💯